### PR TITLE
Update passing remove from fail lists 2023-07-10.

### PR DESCRIPTION
**Pull request by BillyONeal**
```
PASSING, REMOVE FROM FAIL LIST: chipmunk:arm64-android (/vcpkg/scripts/azure-pipelines/../ci.baseline.txt).  
PASSING, REMOVE FROM FAIL LIST: chipmunk:x64-android (/vcpkg/scripts/azure-pipelines/../ci.baseline.txt).  
```

```
REGRESSION: fastrtps:arm-neon-android failed with BUILD_FAILED. If expected, add fastrtps:arm-neon-android=fail to /vcpkg/scripts/azure-pipelines/../ci.baseline.txt.REGRESSION: fastrtps:arm64-android failed with BUILD_FAILED. If expected, add fastrtps:arm64-android=fail to /vcpkg/scripts/azure-pipelines/../ci.baseline.txt. REGRESSION: fastrtps:x64-android failed with BUILD_FAILED. If expected, add fastrtps:x64-android=fail to /vcpkg/scripts/azure-pipelines/../ci.baseline.txt.
```
Previously blocked by:
```
PASSING, REMOVE FROM FAIL LIST: foonathan-memory:arm-neon-android (/vcpkg/scripts/azure-pipelines/../ci.baseline.txt). PASSING, REMOVE FROM FAIL LIST: foonathan-memory:arm64-android (/vcpkg/scripts/azure-pipelines/../ci.baseline.txt). PASSING, REMOVE FROM FAIL LIST: foonathan-memory:x64-android (/vcpkg/scripts/azure-pipelines/../ci.baseline.txt).
```

```
REGRESSION: cmake-user:x86-windows cascaded, but it is required to pass. (C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt).
REGRESSION: hdf5:x86-windows failed with BUILD_FAILED. If expected, add hdf5:x86-windows=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt.
REGRESSION: vcpkg-ci-mathgl:x86-windows cascaded, but it is required to pass. (C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt).
REGRESSION: vcpkg-ci-opencv:x86-windows cascaded, but it is required to pass. (C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt).
REGRESSION: vcpkg-ci-openimageio:x86-windows cascaded, but it is required to pass. (C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt).
REGRESSION: vcpkg-ci-paraview:x86-windows cascaded, but it is required to pass. (C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt).

CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: "C:/Program Files/PowerShell/7/pwsh.exe" -noprofile -executionpolicy Bypass -nologo -file C:/a/1/s/scripts/buildsystems/msbuild/applocal.ps1 -targetBinary D:/packages/hdf5_x86-windows/tools/hdf5/h5clear-shared.exe -installedDir D:/packages/hdf5_x86-windows/bin -verbose
    Working Directory: C:/a/1/s
    Error code: Access violation
    See logs for more information:
      D:\buildtrees\hdf5\copy-tool-dependencies-2-out.log
      D:\buildtrees\hdf5\copy-tool-dependencies-2-err.log
```

Previous CI run passed, so I'm going to hope we got a bad machine or something and move on for now.

```
REGRESSION: openvino:x64-linux failed with BUILD_FAILED. If expected, add openvino:x64-linux=fail to /agent/_work/1/s/scripts/azure-pipelines/../ci.baseline.txt. REGRESSION: openvino:x64-osx failed with BUILD_FAILED. If expected, add openvino:x64-osx=fail to /Users/vagrant/Data/work/2/s/scripts/azure-pipelines/../ci.baseline.txt. REGRESSION: openvino:x64-windows failed with BUILD_FAILED. If expected, add openvino:x64-windows=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt. REGRESSION: openvino:x64-windows-static failed with BUILD_FAILED. If expected, add openvino:x64-windows-static=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt. REGRESSION: openvino:x64-windows-static-md failed with BUILD_FAILED. If expected, add openvino:x64-windows-static-md=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt.
```
See discussion in https://github.com/microsoft/vcpkg/pull/32418 / https://github.com/openvinotoolkit/openvino/issues/18465 -- openvino conflicts with oneDNN because the vcpkg copy is not the same as their forked copy.

```
REGRESSION: physx:x64-linux failed with FILE_CONFLICTS. If expected, add physx:x64-linux=fail to /agent/_work/1/s/scripts/azure-pipelines/../ci.baseline.txt. REGRESSION: physx:x64-windows failed with FILE_CONFLICTS. If expected, add physx:x64-windows=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt. REGRESSION: physx:x64-windows-static failed with FILE_CONFLICTS. If expected, add physx:x64-windows-static=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt. REGRESSION: physx:x64-windows-static-md failed with FILE_CONFLICTS. If expected, add physx:x64-windows-static-md=fail to C:\a\1\s\scripts\azure-pipelines/../ci.baseline.txt.
```
See discussion in https://github.com/microsoft/vcpkg/pull/32432 -- omniverse-physx-sdk was added 5 days ago and conflicts.

```
REGRESSION: libev:arm-neon-android failed with FILE_CONFLICTS. If expected, add libev:arm-neon-android=fail to /vcpkg/scripts/azure-pipelines/../ci.baseline.txt.
REGRESSION: libev:x64-android failed with FILE_CONFLICTS. If expected, add libev:x64-android=fail to /vcpkg/scripts/azure-pipelines/../ci.baseline.txt.
REGRESSION: libev:x64-linux failed with FILE_CONFLICTS. If expected, add libev:x64-linux=fail to /agent/_work/1/s/scripts/azure-pipelines/../ci.baseline.txt.
REGRESSION: libev:x64-osx failed with FILE_CONFLICTS. If expected, add libev:x64-osx=fail to /Users/vagrant/Data/work/2/s/scripts/azure-pipelines/../ci.baseline.txt.
```

Fixed by https://github.com/microsoft/vcpkg/pull/32471

```
REGRESSION: ryu:x64-linux failed with FILE_CONFLICTS. If expected, add ryu:x64-linux=fail to /agent/_work/1/s/scripts/azure-pipelines/../ci.baseline.txt. error: The following files are already installed in /mnt/vcpkg-ci/installed/x64-linux and are in conflict with ryu:x64-linux Installed by bde:x64-linux
debug/lib/libryu.a
    include/ryu/ryu.h
    lib/libryu.a
```

Broken by https://github.com/microsoft/vcpkg/pull/31961, I don't have a fix yet.

### CODE REVIEW

**Comment by Adela0814** [comment]
> ```
> REGRESSION: ryu:x64-linux failed with FILE_CONFLICTS. If expected, add ryu:x64-linux=fail to /agent/_work/1/s/scripts/azure-pipelines/../ci.baseline.txt. error: The following files are already installed in /mnt/vcpkg-ci/installed/x64-linux and are in conflict with ryu:x64-linux Installed by bde:x64-linux
> debug/lib/libryu.a
>     include/ryu/ryu.h
>     lib/libryu.a
> ```
> 
> Broken by #31961, I don't have a fix yet.

I tried to solve this problem by commenting to install the third-party `ryu` in the `bde` source code, but I couldn't find the command to install `ryu` lib, only the header files.

File: ports/chipmunk/vcpkg.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "chipmunk",
   "version": "7.0.3",
-  "port-version": 5,
+  "port-version": 6,
   "description": "A fast and lightweight 2D game physics library.",
   "homepage": "https://github.com/slembcke/Chipmunk2D",
+  "supports": "!(arm & !arm64 & android)",
```
Comment (autoantwort) [inline ports/chipmunk/vcpkg.json:7]: You can also use `arm32`

Comment (dg0yt) [reply ports/chipmunk/vcpkg.json:7]: I discovered this, too, but it is missing in  
https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#platform-expression

Comment (BillyONeal) [reply ports/chipmunk/vcpkg.json:7]: https://github.com/microsoft/vcpkg-docs/pull/108

**Pull request merged**